compysition/actor.py

```diff
@@ -206,9 +205,12 @@ def _loop_send(self, event, queues, check_output=True):
         if check_output and not isinstance(event, self.output):
             raise InvalidActorOutput("Event was of type '{_type}', expected '{output}'".format(_type=type(event), output=self.output))
 
-        if len(queues) > 0:
-            self._send(queues[0], deepcopy(event))
-            map(lambda _queue: self._send(_queue, deepcopy(event)), queues[1:])
+        if isinstance(queues, _InternalQueuePool):
```

Comment:
Honestly, I was afraid of dicts with non queue object values containing a put attribute.  Thinking that could potentially lead to undesired (non error) side effects.  Though the more I think about it, that is an edge case, and one would have to go out of there way to make that happen.  So .. I am now much more inclined to swap isinstance with try as, I agree, pythonically it is better to ask forgiveness.